src/Symfony/Bundle/FrameworkBundle/Command/ContainerDebugCommand.php

```diff
@@ -62,13 +62,18 @@ protected function configure()
                 new InputOption('env-vars', null, InputOption::VALUE_NONE, 'Displays environment variables used in the container'),
                 new InputOption('format', null, InputOption::VALUE_REQUIRED, 'The output format (txt, xml, json, or md)', 'txt'),
                 new InputOption('raw', null, InputOption::VALUE_NONE, 'To output raw description'),
+                new InputOption('deprecations', null, InputOption::VALUE_NONE, 'To output the deprecations generated when compiling and warming the cache'),
```

Comment:
The container no?

Reply:
I didn't write that part but I can make the change you are suggesting :)

```suggestion
                new InputOption('deprecations', null, InputOption::VALUE_NONE, 'To output the deprecations generated when compiling the container and warming up the cache'),
```